acstools/utils_findsat_mrt.py

```diff
@@ -0,0 +1,1253 @@
+'''
+These are various helper routines for findsat_mrt.py
+'''
+
+import numpy as np
+from astropy.modeling import models, fitting
+from astropy.table import Table, vstack
+from skimage import transform
+from astropy.stats import sigma_clip
+import logging
+from astropy.convolution import Gaussian2DKernel
+from astropy.convolution import convolve
+from skimage.transform._warps import warp
+from skimage._shared.utils import convert_to_float
+from warnings import warn
+from multiprocessing import Pool
+import time
+from astropy.nddata import Cutout2D
+from scipy import interpolate
+from astropy.io import fits
+import warnings
+from astropy.utils.exceptions import AstropyUserWarning
+try:
+    import matplotlib.pyplot as plt
+except ImportError:
+    plt = None
+    warnings.warn('matplotlib not found, plotting is disabled',
+                  AstropyUserWarning)
+
+__taskname__ = "utils_findsat_mrt"
+__author__ = "David V. Stark"
+__version__ = "1.0"
+__vdate__ = "06-Dec-2022"
+__all__ = ['_round_up_to_odd', 'merge_tables', 'good_indices',
```

Comment:
Why include the "hidden" functions?

Reply:
Didn't know the standard here. They have been removed, although I have generally modified which are "hidden".